bundles/org.openhab.binding.venstarthermostat/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -114,6 +114,9 @@
 		<item-type>String</item-type>
 		<label>System Mode</label>
 		<description>Current System Operating Mode</description>
+		<tags>
+			<tag>Control</tag>
```

Comment:
Might conflict with rule 2: https://github.com/openhab/openhab-addons/issues/12262

Reply:
Yes, this one is unexpected.